app/controller/InfoController.js

```diff
@@ -336,12 +336,18 @@ SDL.InfoController = Em.Object.create(
       web_app_frame.name = frame_name;
       web_app_frame.id = frame_name;
       web_app_frame.className = "WebEngineFrame";
+      web_app_frame.hidden = false;
 
       const web_engine_view = document.getElementById("webEngineView");
       if (web_engine_view) {
         web_engine_view.appendChild(web_app_frame);
       }
 
+      for(var key in SDL.SDLModel.webApplicationFramesMap) {

```

Comment:
@YarikMamykin not the best place for hiding frames.
I suggest you to add a separate function to `Controller.js` and iterate over `webApplicationFramesMap` there
You may also make this loop simpler:
```
for(var key in SDL.SDLModel.webApplicationFramesMap) {
  SDL.SDLModel.webApplicationFramesMap[key].hidden = key != policyAppID;
}
```

I think the good place to invoke that function is `StateManager.js` (webViewApp -> enter)
You may retrieve `appID` by `SDL.SDLController.model.appID`

Reply:
@AKalinich-Luxoft Reworked in https://github.com/LuxoftSDL/sdl_hmi/pull/39/commits/a6888faaaa3b4974f60ddefc2695c40116eb37a0 & https://github.com/LuxoftSDL/sdl_hmi/pull/39/commits/4cbf29a44478d97cb77c2952c353fc6e722a4ad7